src/js/components/addData/AddDataContent.jsx

```diff
@@ -45,8 +63,29 @@ export default class AddDataContent extends React.Component {
             actionArea = <SubmitButton className="usa-da-button-bigger" buttonText="Upload & Validate CSV files" buttonDisabled={true} />;
         }
 
+        let warning = null;
+        
+        if(this.props.submission.state === 'failed'){
+            warning = <div className='container short'>
+                        <div className={"alert alert-error text-left"} role="alert">
+                            <span className="usa-da-icon error-icon"><Icons.ExclamationCircle /></span>
+                            <div className="alert-header-text">Your upload has failed</div>
+                            <p>Please try again. If the problem persists, contact the service desk.</p>
+                        </div>
+                    </div>;
+        }else if(this.state.upload){
```

Comment:
No props.submission.state doesnt work, it is a template for a submission rather than the actual submission information. its silly, i know but i can't remove it because it breaks something else